website/docs/dependencies/0080-install-amd-drivers.md

```diff
@@ -0,0 +1,43 @@
+﻿# Install AMD GPU Drivers
+Virtual Client has a dependency component that can be added to a workload or monitor profile to install AMD drivers in Linux and Windows systems. The following section illustrates the
+details for integrating this into the profile.
+
+- [AMD Official Drivers Page](https://www.amd.com/en/support)
+- [AMD GPU Drivers SETUP Profile](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/SETUP-GPU-AMDDRIVERS.json)
+
+## Supported Platform/Architectures
+* linux-x64 (Ubuntu)
+* win-x64 ()
+
+## Profile Parameters for Windows
+This dependency component can be used to download the drivers on Windows from a blob storage using the DependencyPackageInstallation. 
+The following section describes the parameters used by the individual component in the profile in Windows:
+
+| **Parameter** | **Required** | **Description**            |                 **Default**                     |
+|---------------|--------------|----------------------------|-------------------------------------------------|
+| PackageName   | Yes          | The logical name of the package that will be registered with the Virtual Client runtime. This name can be used by other profile components to reference the installation parent directory location for Drivers. |  |
+| Scenario      | No           | A name/identifier for the specific component in the profile. This is used for telemetry purposes only with components in dependency sections of the profile (i.e. cannot be used with --scenarios option on the command line). |  |
```

Comment:
GpuModel is required because code is different, unless we can write code to figure out what gpu user has

Reply:
Yeah correct, by mistake missed. Yes regarding detecting GPU, Aditya has this task in pipeline, so will wait for his change to come and then we can update all our GPU specific code accordingly.